sycl/doc/extensions/ITTAnnotations/ITTAnnotations.rst

```diff
@@ -66,6 +66,9 @@ a wrapper.
 Conditional compilation
 -----------------------
 
+Data Parallel C++ compiler automatically instruments user code through
+SPIRITTAnnotations LLVM pass, which is enabled for targets, that natively
```

Comment:
Note, that not everything is being annotated: just barriers, atomics, work item start/finish.

Reply:
Thanks, I've added that bit.